src/components/IspSelect/IspSelect.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../assets/variables";
+
+.IspSelect {
```

Comment:
i'm fine either way.
i like the unambiguousness of this naming - and i can easily search for it without thinking. but yeah, i switch to dashes for most subcomponents. 

so if that feels too disconnected, we can switch to all dashes.